src/ipyradiant/visualization/base.py

```diff
@@ -0,0 +1,59 @@
+import types
+
+import traitlets as T
+
+import ipywidgets as W
+import networkx as nx
+from rdflib import Graph, URIRef
+
+
+class VisBase(W.VBox):
```

Comment:
I would really love some docstrings that at a minimum as a basic description of these classes and what they are used for (trait docs would be great but not 100% necessary). 

e.g. 
```python
class VisBase(W.VBox):
   """ A visualization base class to contain RDF graphs and defaults for rendering them with graph 
        visualization libraries.

        :param graph: and rdflib.graph.Graph object to visualize
        ....
```